client/a8c-for-agencies/controller.tsx

```diff
@@ -1,9 +1,75 @@
-import { type Callback } from '@automattic/calypso-router';
+import page, { type Callback } from '@automattic/calypso-router';
+import { category, chevronLeft, home } from '@wordpress/icons';
+import Sidebar from './components/sidebar';
 
-export const a8cForAgenciesContext: Callback = ( context, next ) => {
-	context.header = <div>Header</div>;
-	context.secondary = <div>Secondary</div>;
-	context.primary = <div>Hello, World!</div>;
+export const redirectToOverviewContext: Callback = () => {
+	page( '/overview' );
+};
+
+export const overviewContext: Callback = ( context, next ) => {
+	context.secondary = (
+		<Sidebar
+			path="/"
+			menuItems={ [
+				{
+					icon: home,
+					path: '/',
+					link: '/overview',
+					title: 'Overview',
+					trackEventProps: {
+						menu_item: 'A4A / Overview',
+					},
+					isSelected: true,
```

Comment:
isSelected to set to true for this menu item. Are we planning to update this later?

Reply:
see my comment above.